profile/README.md

```diff
@@ -4,3 +4,7 @@
     <img src="https://raw.githubusercontent.com/mojaloop/.github/main/profile/mojaloop_logo.png"/>
   </a>
 </p>
+# Welcome to the Mojaloop Open Source Site on Github
+
+* Review the documentation: http://mojaloop.io/documentation
+* Join the Conversation on Slack: https://mojaloop.io/slack
```

Comment:
Can we maybe put this back when the herokuapp is working?

Reply:
until then we can use this instead: https://join.slack.com/t/mojaloop/shared_invite/zt-1qy6f3fs0-xYfqfIHJ6zFfNXb0XRpiHw